app/api/routes.py

```
@@ -475,4 +477,50 @@ def end_chat_session(request: SessionEndRequest, current_user: User = Depends(Au
+        raise HTTPException(status_code=404, detail="종료할 활성화된 세션을 찾을 수 없습니다.")
+
+
+@router.get(
+    "/api/v1/consult/history",
+    tags=["에이전트 상담"],
+    summary="전체 상담 기록 목록 조회",
+    description="세션별 상담 이력을 하나씩 묶어 전체 상담 기록 목록을 조회합니다.",
+    response_model=list[ConsultSessionSummary],
+)
+def get_consult_history_routes(
+    db: Session = Depends(get_db),
```

Ruff (0.14.5)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)